STP-3402: Change default recipe values in integration (#54)
Cray-HPE/docs-sat: docs/usage/sat_bootprep.md

+----------------+------------+---------+---------------------+
```

To overwrite the desired configuration using `sat bootprep`, ensure the bootprep
input file specifies to create a configuration with the same name
(`ncn-personalization` in the example above). To create a different configuration,
ensure the bootprep input file specifies to create a configuration with a

> different name than the desired configuration. In the example above, you should

Reviewer: > In the example above, you should not specify creating a configuration named `ncn-personalization` to avoid overwriting the desired configuration.

I still feel like this is not totally clear that you have the choice of whether or not to overwrite the current config, seems more like it is specifically telling you to avoid overwriting. After a couple reads it made sense to me though, so maybe there's nothing to fix here.

Reviewer 2: I had the same first impression reading this final sentence. To make clear that overwriting is optional, maybe something like the following would convey that? I'm not totally satisfied with the negation at the end of my second sentence, I think it would need more workshopping.

```
In the example above, if you want to avoid overwriting the desired configuration,
you should specify creating a configuration with a name other than `ncn-personalization`.
```

Author: @ekamin-hpe and @jack-stanek-hpe, I don't know why this is my first time seeing this comment. What if we said something like the following instead of lines 608 - 610?

```suggestion
different name than the desired configuration (different from `ncn-personalization` in the example above).
```
